mirage/scenarios/default.ts

```diff
@@ -17,14 +17,7 @@ const {
 } = config;
 
 export default function(server: Server) {
-    let currentUser = null;
-
-    if (defaultLoggedOut) {
-        currentUser = server.create('user');
-        server.create('root', { currentUser: null });
-    } else {
-        currentUser = server.create('user', 'loggedIn');
-    }
+    const currentUser = defaultLoggedOut ? server.create('user') : server.create('user', 'loggedIn');
```

Comment:
`currentUser: null` is important unless you fixed the pages that have a problem with not finding a user. Can I remember which pages? I can not.

Reply:
I didn't fix those pages, but it's not necessary to create a root here at all. If there's no user with the `loggedIn` trait, the [root created by default has `currentUser: null`](https://github.com/CenterForOpenScience/ember-osf-web/blob/ca61f3448da94e039ea2d6126a16003d8f22452a/mirage/views/root.ts#L4).

Reply:
Very good. Late November is probably after the last time I saw the problem 👍